### ISSUE TITLE
loading new yang file is not success

### ISSUE THREAD

**Issue by ajan2021** [CESNET/netopeer2#798]
Hi,
New to netopeer2 and tried to install sample yang module but it is in N state in sysrepoctl -l output and after docker stop/start the files under /data directory are no more available.please let me know whether there is any other process to load yang file in netopeer2 apart from sysrepo.
----
root@f3d8e82b8422:/etc/sysrepo/yang# sysrepoctl -i /etc/sysrepo/yang/example-1.yang
root@f3d8e82b8422:/etc/sysrepo/yang# sysrepoctl -l
Sysrepo repository: /etc/sysrepo

Module Name                | Revision   | Flags | Owner     | Permissions | Submodules | Features                                                               
----------------------------------------------------------------------------------------------------------------------------------------------------------------
example-1                  |            |  N    |           |             |            |                                                                        
iana-crypt-hash            | 2014-08-06 | i     |           |             |            |                                                                        
ietf-crypto-types          | 2019-07-02 | I     | root:root | 600         |            |                                                                        
ietf-datastores            | 2018-02-14 | I     | root:root | 666         |            |                                                                        
ietf-inet-types            | 2013-07-15 | i     |           |             |            |                                                                        
ietf-keystore              | 2019-07-02 | I     | root:root | 600         |            | keystore-supported                                                     
ietf-netconf               | 2013-09-29 | I     | root:root | 600         |            | writable-running candidate rollback-on-error validate startup url xpath

Thanks

**Comment by michalvasko** [comment]
Run `sysrepoctl -l -v3` and you will see the problem.

**Comment by ajan2021** [comment]
Hi,Below is the output but there is only one connection as of now.

root@f3d8e82b8422:/etc/sysrepo/yang# sysrepoctl -l -v3
[INF]: Scheduled changes not applied because of other existing connections.
Sysrepo repository: /etc/sysrepo

Module Name                | Revision   | Flags | Owner     | Permissions | Submodules | Features                                                               
----------------------------------------------------------------------------------------------------------------------------------------------------------------
example-1                  |            |  N    |           |             |            |                                                                        
iana-crypt-hash            | 2014-08-06 | i     |           |             |            |                                                                        
ietf-crypto-types          | 2019-07-02 | I     | root:root | 600         |            |                                                                        
ietf-datastores            | 2018-02-14 | I     | root:root | 666         |            |                                                                       

------------------------------
root@f3d8e82b8422:/etc/sysrepo/yang# sysrepoctl --connection-count
1
root@f3d8e82b8422:/etc/sysrepo/yang#

**Comment by michalvasko** [comment]
There can be no connection is the module is to be installed. You probably have *netopeer2* running, do you not? You must restart it, at least, so that the module can be installed.

**Comment by ajan2021** [comment]
Yes,netopeer2 was running and module got installed after restart.Thanks much for quick responses.
Also please let me know the below as could not find the Netopeer2 documentation
1.Is there any Java api support for netopeer2 to do config changes from a java application
2. Can i use Netopeer2 instead of ConfD like replacement 
Thanks

**Comment by michalvasko** [comment]
1. There is no Java support (bindings) in any of the current applications. However, you can always use a Java NETCONF client to connect to *netopeer2* and send whatever RPCs (including `edit-config`) you want.
2. Yes, I would say so. But I do not know much about ConfD.
